pkg/vault/config.go

```diff
@@ -36,9 +36,10 @@ func NewConfig() (*Config, error) {
 		VaultAPIClient: apiClient,
 	}
 
+	auth := getEnv("AVP_AUTH_TYPE", "")
+
 	switch getEnv("AVP_TYPE", "") {
 	case "vault":
```

Comment:
I also wonder now if these strings (the values for these env variables) should be constants, since they are kind of like enums right?

Reply:
Can pull out into consts

Reply:
Actually.. i will create an issue to refactor so we can get this in